lib/presentation/shared_widgets/micro_crowdaction_card.dart

```diff
@@ -0,0 +1,92 @@
+import 'package:auto_route/auto_route.dart';
+import 'package:flutter/material.dart';
+
+import '../../domain/crowdaction/crowdaction.dart';
+import '../routes/app_routes.gr.dart';
+import '../themes/constants.dart';
+import 'accent_chip.dart';
+
+class MicroCrowdActionCard extends StatelessWidget {
+  final CrowdAction crowdAction;
+  const MicroCrowdActionCard(this.crowdAction, {Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return GestureDetector(
+      onTap: () => context.router
+          .push(CrowdActionDetailsRoute(crowdAction: crowdAction)),
+      child: Container(
+        height: 148,
+        width: MediaQuery.of(context).size.width,
+        margin: const EdgeInsets.symmetric(vertical: 10),
+        child: Card(
+          shape: RoundedRectangleBorder(
+            borderRadius: BorderRadius.circular(20),
+          ),
+          elevation: 4,
+          child: Row(
+            children: [
+              Container(
+                decoration: BoxDecoration(
+                  color: Colors.black,
+                  borderRadius: BorderRadius.circular(15),
+                  image: DecorationImage(
+                    image: NetworkImage(crowdAction.images.card),
+                    fit: BoxFit.cover,
+                  ),
+                ),
+                margin: const EdgeInsets.only(left: 10),
+                height: 128,
+                width: 100,
+              ),
+              Expanded(
+                child: Padding(
+                  padding: const EdgeInsets.symmetric(horizontal: 10),
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: [
+                      const AccentChip(
+                        text: "Sign up now",
```

Comment:
TODO: Only show for crowdactions where the "join deadline" has not expired. (Mark with TODO for MVP)

Reply:
Maybe we can actually already implement this like so `!crowdaction.dateLimitJoin.isAfter(DateTime.now())`.

Reply:
Done, will be in next commit